build.xml

```diff
@@ -18,22 +21,73 @@
+         com/zimbra/oauth/schedulers/impl/**,
+         com/zimbra/oauth/managers/ClassManager.class,
+         com/zimbra/oauth/utilities/**"/>
+     </antcall>
+   </target>
+
+  <target name="oauth-social-common-jar" depends="compile"
+        description="Creates the zm-oauth-social-common.jar file which is used by other extensions">
+      <antcall target="zimbra-jar">
+        <param name="implementation.title" value="Zimbra OAuth Social Common Jar"/>
+        <param name="jar.file" value="${oauth-social-common.jar.file}"/>
+        <param name="includes"
+          value="com/zimbra/oauth/cache/**,com/zimbra/oauth/token/**,
+          com/zimbra/oauth/handlers/**,com/zimbra/oauth/models/**,
+          com/zimbra/oauth/schedulers/**,
+          com/zimbra/oauth/utilities/OAuth2HttpConstants.class, 
+          com/zimbra/oauth/utilities/OAuth2CacheUtilities.class,
+          com/zimbra/oauth/utilities/OAuth2Constants.class,
+          com/zimbra/oauth/utilities/OAuth2DataSource*.class,
+          com/zimbra/oauth/utilities/OAuth2JsonUtilities.class,
+          com/zimbra/oauth/utilities/OAuth2Utilities.class"/>
+        <param name="excludes"
+          value="com/zimbra/oauth/handlers/impl/**,
+          com/zimbra/oauth/schedulers/impl/**"/>
+
+      </antcall>
+    </target>
+
   <!-- deploy to the server extension directory -->
   <target name="deploy" depends="jar">
     <delete dir="${extension.deploy.dir}"/>
     <mkdir dir="${extension.deploy.dir}"/>
     <ivy:install organisation="com.auth0" module="java-jwt" revision="3.2.0" settingsRef="dev.settings" from="chain-resolver" to="build-dist" overwrite="true" transitive="true" type="jar"/>
-    <ivy:install organisation="com.fasterxml.jackson.core" module="jackson-annotations" revision="2.10.1" settingsRef="dev.settings" from="chain-resolver" to="build-dist" overwrite="true" transitive="true" type="jar"/>
-    <ivy:install organisation="com.fasterxml.jackson.core" module="jackson-core" revision="2.10.1" settingsRef="dev.settings" from="chain-resolver" to="build-dist" overwrite="true" transitive="true" type="jar"/>
-    <ivy:install organisation="com.fasterxml.jackson.core" module="jackson-databind" revision="2.10.1" settingsRef="dev.settings" from="chain-resolver" to="build-dist" overwrite="true" transitive="true" type="jar"/>
+    <ivy:install organisation="com.fasterxml.jackson.core" module="jackson-annotations" revision="2.9.2" settingsRef="dev.settings" from="chain-resolver" to="build-dist" overwrite="true" transitive="true" type="jar"/>
+    <ivy:install organisation="com.fasterxml.jackson.core" module="jackson-core" revision="2.9.2" settingsRef="dev.settings" from="chain-resolver" to="build-dist" overwrite="true" transitive="true" type="jar"/>
+    <ivy:install organisation="com.fasterxml.jackson.core" module="jackson-databind" revision="2.9.2" settingsRef="dev.settings" from="chain-resolver" to="build-dist" overwrite="true" transitive="true" type="jar"/>
```

Comment:
any specific reason to downgrade the version.

Reply:
Good catch, Fixed